.github/workflows/tests.yml

```diff
@@ -21,8 +21,8 @@ jobs:
       matrix:
         os: ['ubuntu-latest', 'macos-latest']
         python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
-        cratedb-version: ['5.1.2']
-        sqla-version: ['1.3.24', '1.4.45']
+        cratedb-version: ['5.2.0']
```

Comment:
Maybe 5.1.3 or 5.2.1 ? (I think a decision was made to not promote 5.2.0  to stable)

Reply:
Thanks. I will update to CrateDB 5.2.1 right away when there will be a tarball.